index.js

```diff
@@ -8,7 +8,9 @@ const run = async () => {
 }
 
 try {
-  run()
+  run().catch((error) => {
+    core.setFailed(`Action failed with error: ${error}`)
+  })
 } catch (error) {
```

Comment:
IMHO it's safer to still have the original `catch` just in case anything goes wrong at https://github.com/zendesk/setup-jsonnet/blob/b28175da22259a03fa0fb54e221dae6b5dc6d72a/index.js#L6 (https://github.com/zendesk/setup-jsonnet/blob/b28175da22259a03fa0fb54e221dae6b5dc6d72a/node_modules/@actions/core/lib/core.js#L93-L95).

WDYT? @Sancerio

Reply:
Ah, But those are also inside the async so it will be catch by the `.catch()` too. @Frederick888 

Actually I was more inclined to do something like:
```js
const run = async () => {
  try {
  	const version = core.getInput('version')
  	await exec(path.join(__dirname, 'install-jsonnet.sh'), version)
  } catch (error) {
    ...
  }
}

run()
```

as it is cleaner than the lambda style promise.

Reply:
Ah yup, you were right. Thanks for the suggestion, I've just updated PR :)